Source/Core/VideoBackends/OGL/PostProcessing.cpp

```diff
@@ -27,7 +27,7 @@ static const char s_vertex_shader[] =
 	"void main(void) {\n"
 	"	vec2 rawpos = vec2(gl_VertexID&1, gl_VertexID&2);\n"
 	"	gl_Position = vec4(rawpos*2.0-1.0, 0.0, 1.0);\n"
-	"	uv0 = rawpos * src_rect.zw + src_rect.xy;\n"
+	"	uv0 = vec2(mix(src_rect.xy, src_rect.zw, rawpos));\n"
```

Comment:
Yes, but the constant values are also changes in the next lines. My question was about moving CPU operations to per-pixel GPU operations ;)